packages/lesswrong/components/common/CloudinaryImage2.tsx

```diff
@@ -40,6 +52,8 @@ const CloudinaryImage2 = ({width, height, objectFit, publicId, className}: {
   if (objectFit) {
     imageStyle.objectFit = objectFit
   }
+  
+  Object.assign(cloudinaryProps, imgProps)
```

Comment:
```suggestion
  cloudinaryProps = {...cloudinaryProps, ...imgProps}
```

I'd prefer (/ it would match most uses in this codebase) to use object spreading instead of Object.assign